instrumentation/openai/openai-java-1.1/library/src/main/java/io/opentelemetry/instrumentation/openai/v1_1/InstrumentedChatCompletionService.java

```diff
@@ -85,4 +99,39 @@ private ChatCompletion createWithLogs(
     ChatCompletionEventsHelper.emitCompletionLogEvents(eventLogger, result, captureMessageContent);
     return result;
   }
+
+  public StreamResponse<ChatCompletionChunk> createStreaming(
```

Comment:
other methods are private, does it have to be public or is it a typo?

Reply:
Oops, thanks